src/gen_types.py

```diff
@@ -165,7 +165,6 @@ def add_type(name, sz):
  * List of all built-in types.
  */
 enum GPUARRAY_TYPES {
-  GA_POINTER = -2,
```

Comment:
We can bump the API version but that would entail also bumping Theano since it checks for version 1.